packages/core/src/components/tabs/tabs.tsx

```diff
@@ -285,9 +293,8 @@ export class Tabs extends AbstractPureComponent2<TabsProps, ITabsState> {
 
         let indicatorWrapperStyle: React.CSSProperties = { display: "none" };
         if (selectedTabElement != null) {
-            const { clientHeight, clientWidth, offsetLeft, offsetTop } = selectedTabElement;
```

Comment:
`clientHeight` is not needed thanks to `align-self: stretch;` on the tab

Reply:
it looks like we still need this for vertical tabs.